src/lib/mongoBackend/safeBsonGet.cpp

```diff
@@ -105,8 +102,31 @@ int getIntField(const BSONObj& b, const std::string& field)
 
 /* ****************************************************************************
 *
-* getBoolField -
+* getLongField -
+*/
+long getLongField(const BSONObj& b, const std::string& field)
```

Comment:
I guess we could.
It will get a little slower in run-time, as more checks are made.
I think I would prefer to *always* use **Long**, and never use **Int** instead ...